presto-hive-common/src/main/java/com/facebook/presto/hive/filesystem/DirectoryLister.java

```diff
@@ -11,18 +11,24 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package com.facebook.presto.hive;
+package com.facebook.presto.hive.filesystem;
```

Comment:
You don't need to move `DirectoryLister` to presto-hive-common, right? Just keep it in presto-hive.

Reply:
Yeah, most of the refactoring came because of directorylister changes.
However the reason I didn't move back, because our initial idea was to keep most of the common models to hive-common.

Reply:
`presto-hive-common` will become too thick. Let's only move necessary classes to this module. `DirectoryLister` can now live in `presto-hive`

Reply:
ok.